cluster/config.go

```diff
@@ -36,6 +36,18 @@ func getKesContainerImage() string {
 	return env.Get(kesImage, "minio/kes:latest")
 }
 
+func getKesRunningPort() string {
```

Comment:
Should we do input validation here?
Like checking that the port is a numder (uint16) and return an int instead of a string?